test/functional/executor.py

```diff
@@ -62,8 +62,10 @@ def run(self):
                 print("TEST: ", test.get_name())
                 print("DESCRIPTION: ", test.get_description())
                 print("RUNNING: ", *cmd)
-            subprocess.run(cmd, stdout=subprocess.DEVNULL,
-                           stderr=subprocess.DEVNULL)
+                print(subprocess.getoutput(cmd))
```

Comment:
We may address this issues in other task if you like. Overall, this can be merged.